contents/handbook/people/team-structure/core-experience.md

```diff
@@ -24,51 +27,48 @@ organization in matters of product design, user experience, and frontend & API t
 - Maintain a coherent strategy for frontend and API development (technologies, tools and methodology).
-- Always be in touch with product leadership and our customers, in order to figure out what works, what doesn't, and what needs to be built next.
-- Actually build those features.
-- Support other teams in PostHog in maintaining a coherently high level of product experience across the board.
-- Push the envelope and give back to the open source community whenever we develop abstractions that could benefit other organizations.
+- Ensuring our features are used the way we expect them to.
+- Understanding users and their needs... and solving for that.
 
 ## Goals
 
-With the work above, we aim to solve the following meta-goals:
-
-- Part of our vision: It should be unbelievably easy and intuitive for anyone to use PostHog to get answers to  questions they have, and even find the answers to questions they had not considered asking.
+What is quality and how do we track it?
 
-- To make our tech worthy of the fanatical following that is emerging around PostHog.
-- Build the next generation of engineers and engineering leaders, who would be running things when we are at 100 developers.
-- Remain a highly flexible team, ready to adapt to changing business requirements.
-- Be part of formalizing "the PostHog way", and writing a book about it.
-- Our tech stack is rather unique, though pretty powerful. We are also in a really powerful position to drive change in the industry ([data-first frontend](https://kea.js.org/blog/data-first-frontend-revolution), anyone?). Let's capitalise on that.
-- Zero TypeScript errors, code covered by logic tests, all scenes in Storybook, etc.
-- To do a damn good job at all of the above.
+We track specific goals/targets depending on feature. For example:
 
-In short, if you want a crash course at how to build large and complex webapps, at scale, while retaining a maintainable codebase and high customer satisfaction, and touch every part of the stack, this is the team to work with.
+- Qualitative data (what users are saying)
+- Measured data (KPIs, performance metrics, etc)
+- Internal company feedback (meeting company goals)
 
+Main metric: number of discoveries.
 
+## Scope
 
-## Responsibilities
-List below is prioritized.
+We handle everything not owned by Core Analytics and Platform.
 
-1. Reaching parity in terms of features, performance and user experience with leading product analytics solutions, including:
-   1. Extending core feature set (particularly on the frontend) as informed by dogfooding, user feedback, experimentation and strategic decisions.
-   2. Extending user experience to enable product analytics for non-technical users. 
-2. Maintaining (and improving where applicable) supporting activities for the core experience (this includes: cohorts, persons, sessions *[not session recording]*, annotations, settings, authentication, account management). 
+We handle everything people in the team have worked on.
```

Comment:
This is basically saying we own all the stuff we built or touched recently. I think there's some snark in here, but it does bring up the issue of who owns what after it's built.

Reply:
Hmm not 100% sure about this. For instance, I could see the case of CoreAn taking over the querying part of recordings, or CoreExp taking over the interaction of Paths/Funnels with Recordings. While generally this would be true, I wouldn't word it as a hard rule. Maybe as below? But still unsure it adds additional value.

```suggestion
- We own (maintain and improve) features we've built, unless there's a total/partial hand-off to another team?
```